services/director-v2/src/simcore_service_director_v2/modules/dask_clients_pool.py

```diff
@@ -102,30 +99,20 @@ async def _concurently_safe_acquire_client() -> DaskClient:
 
         try:
             dask_client = await _concurently_safe_acquire_client()
-            yield dask_client
-        except (
-            MissingComputationalResourcesError,
-            InsuficientComputationalResourcesError,
-        ):
-            raise
-        except (
-            asyncio.CancelledError,
-            ComputationalBackendNotConnectedError,
-            ComputationalSchedulerChangedError,
-        ):
-            # cleanup and re-raise
-            if dask_client := self._cluster_to_client_map.pop(cluster.id, None):
-                await dask_client.delete()
-            raise
         except Exception as exc:
-            # cleanup and re-raise
-            if dask_client := self._cluster_to_client_map.pop(cluster.id, None):
-                await dask_client.delete()
-            logger.error(
-                "could not create/access dask computational cluster %s",
-                json_dumps(cluster),
-            )
             raise DaskClientAcquisisitonError(cluster=cluster, error=exc) from exc
+        else:
+            try:
+                yield dask_client
+            except (
```

Comment:
so now every other exception is just "let-go" ... who is handling it then?

Reply:
the scheduler